### PR TITLE
[docs-only]adding documemtation for graph api (Permissions)

### PR DESCRIPTION
**Pull request by ghost**
## Description

This PR adds documentation for permissions endpoint

## Related Issue

- Relate issue:https://github.com/owncloud/ocis/issues/3139

## How Has This Been Tested?

- test environment:
- test case 1:
- test case 2:
- ...

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Review by phil-davis — APPROVED** [review]
LGTM
@ScharfViktor @saw-jan please also review.

File: docs/apis/http/graph/permissions.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        "libre.graph/driveItem/path/update"
+      ]
+    }
+  ]
+}
+````
+
+## Creating Permissions / Sharing
+
+### Create a sharing link for a driveitem `POST /drives/{drive-id}/items/{item-id}/createLink`
```
Comment (saw-jan) [inline docs/apis/http/graph/permissions.md:154]:
```suggestion
### Create a link share `POST /drives/{drive-id}/items/{item-id}/createLink`
```

File: docs/apis/http/graph/permissions.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+https://owncloud.dev/libre-graph-api/#/drives.permissions/CreateLink
+
+### Send a sharing invitation `POST /drives/{drive-id}/items/{item-id}/invite`
```
Comment (saw-jan) [inline docs/apis/http/graph/permissions.md:158]:
```suggestion
### Create a user/group share `POST /drives/{drive-id}/items/{item-id}/invite`
```

**Review by saw-jan — CHANGES_REQUESTED** [review]

File: docs/apis/http/graph/permissions.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+      "@libre.graph.permissions.actions": [
+        "libre.graph/driveItem/basic/read",
+        "libre.graph/driveItem/path/update"
+      ]
+    }
+  ]
+}
+````
+
+## Creating Permissions / Sharing
```
Comment (saw-jan) [inline docs/apis/http/graph/permissions.md:152]: should we only call shares or permissions
CC @ScharfViktor 
```suggestion
## Creating Shares
```

File: docs/apis/http/graph/permissions.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+https://owncloud.dev/libre-graph-api/#/drives.permissions/Invite
+
+## Reading Permissions
+
+### List the effective sharing permissions on a driveitem `GET /drives/{drive-id}/items/{item-id}/permissions`
+
+https://owncloud.dev/libre-graph-api/#/drives.permissions/ListPermissions
+
+### List Get sharing permission for a file or folder `GET /drives/{drive-id}/items/{item-id}/permissions/{perm-id}`
```
Comment (saw-jan) [inline docs/apis/http/graph/permissions.md:168]: I think, this endpoint is to list a single share of any item not the permissions of a file or folder.
```suggestion
### List Get sharing permission for a file or folder `GET /drives/{drive-id}/items/{item-id}/permissions/{perm-id}`
```

File: docs/apis/http/graph/permissions.md (hunk 3 above)
Comment (ScharfViktor) [reply docs/apis/http/graph/permissions.md:152]: I would stick with the naming from the api spec. 
Maybe: `Creating Share Invitation / Link` ?

**Review by v-scharf — COMMENTED** [review]
LGTM

File: docs/apis/http/graph/permissions.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+
+https://owncloud.dev/libre-graph-api/#/drives.permissions/CreateLink
+
+### Create a user/group share `POST /drives/{drive-id}/items/{item-id}/invite`
+
+https://owncloud.dev/libre-graph-api/#/drives.permissions/Invite
+
+## Reading Permissions
+
+### List the effective sharing permissions on a driveitem `GET /drives/{drive-id}/items/{item-id}/permissions`
```
[the remark this answers is not in the document]
Comment (saw-jan) [reply docs/apis/http/graph/permissions.md:164]: driveitem means files/folders in a drive

**Review by saw-jan — APPROVED** [review]

**Pull request merged**